packages/truffle-artifactor/test/contracts.js

```diff
@@ -51,94 +49,94 @@ describe("artifactor + require", function() {
     };
 
     // Compile first
-    var result = await promisify(Compile)(sources, options);
+    const result = await promisify(Compile)(sources, options);
 
     // Clean up after solidity. Only remove solidity's listener,
     // which happens to be the first.
     process.removeListener(
       "uncaughtException",
-      process.listeners("uncaughtException")[0] || function() {}
+      process.listeners("uncaughtException")[0] || (() => {})
     );
 
-    var compiled = Schema.normalize(result["Example"]);
+    const compiled = Schema.normalize(result["Example"]);
     abi = compiled.abi;
-    binary = compiled.bytecode;
+    bytecode = compiled.bytecode;
 
     // Setup
-    var dirPath = temp.mkdirSync({
+    const dirPath = temp.mkdirSync({
       dir: path.resolve("./"),
       prefix: "tmp-test-contract-"
     });
 
-    var expected_filepath = path.join(dirPath, "Example.json");
+    const expected_filepath = path.join(dirPath, "Example.json");
 
     artifactor = new Artifactor(dirPath);
 
     await artifactor
       .save({
         contractName: "Example",
-        abi: abi,
-        binary: binary,
-        address: "0xe6e1652a0397e078f434d6dda181b218cfd42e01",
-        network_id: network_id
+        abi,
+        bytecode,
+        networks: {
+          [`${network_id}`]: {
+            address: "0xe6e1652a0397e078f434d6dda181b218cfd42e01"
+          }
+        }
       })
-      .then(function() {
-        var json = requireNoCache(expected_filepath);
+      .then(() => {
+        const json = requireNoCache(expected_filepath);
         Example = contract(json);
         Example.setProvider(provider);
       });
   });
 
-  before(function() {
-    return web3.eth.getAccounts().then(accs => {
+  before(() =>
+    web3.eth.getAccounts().then(accs => {
```

Comment:
I think we shouldn't use abbreviations. I would like to change **accs** to **accounts**.